easybuild/easyconfigs/n/novoalign/novoalign-3.09.01-foss-2018b-R-3.5.1.eb

```diff
@@ -0,0 +1,58 @@
+easyblock = 'PackedBinary'
+
+name = 'novoalign'
+version = '3.09.01'
+versionsuffix = '-R-%(rver)s'
```

Comment:
Not sure that the -R suffix is useful here.

Reply:
I would rather keep this here, as `novoaligin` is a dependency, and I do not want to change them as well. At the end it does not hurt.